forge_sync/resources.py

```diff
@@ -0,0 +1,20 @@
+import os
+from typing import Dict
+
+import pymongo
+from dagster import ConfigurableResource, Definitions
+
+
+mongo_client = pymongo.MongoClient(os.getenv("MONGO_URI"))
```

Comment:
É interessante termos uma rotina para inicialização dessas variáveis de ambiente.

Reply:
Não entendi. Pipeline pra pegar variável de ambiente?

Reply:
Se você colocar as variáveis no file .env, elas sobem automático.

Reply:
Não precisa de pipeline, mas de algo que inicie as variáveis. As variáveis não sobem automaticamente, isso é uma feature do pipenv. É bom deixar explícito. O python tem um package pra isso: https://pypi.org/project/python-dotenv/.

Reply:
`pipenv` tem mas não to usando o dele 😄 . É feature do dagster:
https://docs.dagster.io/guides/dagster/using-environment-variables-and-secrets#using-environment-variables-and-secrets

Reply:
Beleza então. Só vamos deixar no readme que o arquivo env está sendo usando como template.